explainer.md

```diff
@@ -52,6 +52,9 @@ for (item in details) {
 }
 ```
 
+
+The returned `itemDetails` sequence may be in any order and may not include items if information was not found on the server.
```

Comment:
How about "may not include an item if it doesn't exist on the server. (i.e., there is not a 1:1 correspondence between the input list and output)"?

The parenthetical part is important to call out this design point, to give people an opportunity to object.

Reply:
Done.